Library/Homebrew/formula.rb

```diff
@@ -2620,6 +2623,23 @@ def test(&block)
       define_method(:test, &block)
     end
 
+    # Livecheck can be used to check if there are any updates to the Formula.
+    # WIP
+    def livecheck(&block)
+      instance_eval(&block)
+    end
+
+    # Handles the `skip` feature for the livecheck Formula DSL
+    # WIP
+    def skip(skip_msg)
+      skip_msg = if skip_msg.is_a?(String) && !skip_msg.empty?
+        " - #{skip_msg}"
+      else
+        ""
+      end
+      "skipped#{skip_msg}"
+    end
```

Comment:
```suggestion
    def skip(skip_msg = nil)
      @skipped = true
      @skip_msg = skip_msg
    end
```

I think it would be simpler and more in keeping with other methods in `formula.rb` if `skip` simply set a `@skipped` instance variable to `true` (indicating that `skip` was present in the `livecheck` block in the formula) and set a `@skip_msg` instance variable to the provided message or `nil` by default.

I think it's more appropriate to handle the output for the "skipped" text and printing `skip_msg` in  Homebrew/livecheck instead. The trouble is that we can only do that if we make `@skipped` and `@skip_msg` available to livecheck through public methods (e.g., `skipped?` and `skip_msg`). From my testing, this would seemingly also require adding `skipped?` and `skip_msg` methods to `extend/formula.rb` in Homebrew/livecheck (which simply call the class methods).

This bumps into the same issue I had before while working on the version-filtering feature, where livecheck's access is limited and we seemingly have to expose things that we may not want to expose in `formula.rb`. If it's not appropriate to expose `skipped?` and `skip_msg` methods, we would seemingly have to wait until after Homebrew/livecheck is part of Homebrew/brew to do add implementation for `skip` there.

Reply:
Thanks for the suggestion, @samford. I did run into the issue of calling/using private methods when I was trying out different implementations. For now, I guess, I'll go ahead and make this change.